### PR TITLE
add ability to send a blank email to contacts

### PR DESCRIPTION
**Pull request by ssoonmi**
Screen Shot 2019-09-14 at 4 58 11 AM

### CODE REVIEW

**Comment by anjosh** [comment]
Thank you! Could you change the wording to "**Write** your own email"?

**Comment by ssoonmi** [comment]
changed to "Write your own email"

Also changed mailto: string creation so contacts with no email don't show up as 'null' in the string

File: pages/content.vue [hunk 1]
```diff
@@ -229,9 +269,9 @@ Sincerely,`
           if (this.$route.params.people) {
             const {people} = this.$route.params
             if (people.length === 1) {
-              mailto = people[0].email + '?'
+              mailto = people[0].email || '' + '?'
             } else if (people.length > 1) {
-              mailto = '?bcc=' + people.map(person => person.email).join(',') + '&'
+              mailto = '?bcc=' + people.map(person => person.email || '').join(',') + '&'
```
Comment (anjosh) [inline pages/content.vue]: I tested sending to multiple contacts without this change and I don't see any nulls. .map is handling that for us. So let's remove this line.

**Comment by ssoonmi** [comment]
done

File: pages/content.vue [hunk 2]
```diff
@@ -66,6 +66,36 @@
                 </v-layout>
               </v-card>
             </v-flex>
+            <v-flex xs12>
+              <v-layout align-center row>
+                <v-flex><v-divider></v-divider></v-flex>
+                <v-flex 
+                  style="max-width: fit-content;"
```
Comment (anjosh) [inline pages/content.vue]: Some more style

**Pull request merged**